primitives/api/src/lib.rs

```diff
@@ -734,5 +734,16 @@ decl_runtime_apis! {
 	pub trait Metadata {
```

Comment:
```suggestion
	#[api_version(2)]
	pub trait Metadata {
```

Not that we forget this! :D